pangeo_forge/tasks/xarray.py

```diff
@@ -1,43 +1,69 @@
+from typing import List
+
 import fsspec
 import xarray as xr
 from prefect import task
 
 
 @task
-def combine_and_write(sources, target, append_dim, concat_dim, first=True):
+def combine_and_write(sources: List[str], target: str, concat_dim: str) -> List[str]:
```

Comment:
Ah... I see I've mixed together `concat_dim` and `append_dim`. I'll look into why they were separate.